tools/tests/test_lint.py

```
@@ -1144,3 +1456,313 @@ def test_non_mapping_dispositions_and_staffing_are_findings(tmp_path):
+def test_cell_frontmatter_fires_when_the_name_disagrees_with_the_directory(tmp_path):
+    make_clean_tree(tmp_path)
+    cell = tmp_path / "skills" / "charter" / "SKILL.md"
+    cell.write_text(
+        cell.read_text(encoding="utf-8").replace("name: charter", "name: chartr"),
+        encoding="utf-8",
+    )
+    findings = [f for f in lint.run(tmp_path) if "cell-frontmatter" in f]
+    assert len(findings) == 1 and "sits in" in findings[0]
+
+
+def test_the_declared_description_ceiling_is_the_one_these_tests_pin(tmp_path):
```

Ruff (0.16.2)

[warning] Unused function argument: `tmp_path`

(ARG001)